models/marts/dim_customers.sql

```diff
@@ -13,10 +42,23 @@ final as (
         customers.customer_zip_code_prefix,
         customers.customer_city,
         customers.customer_state,
+        days_since_first_last.days_since_first_order,
+        days_since_first_last.days_since_last_order,
+        days_since_first_last.days_between_first_last_order,
+    --boolean
+        --customer considered active if they have made an order in the past 45 days
+        case 
+            when days_since_first_last.days_since_last_order <= 45 then 1
+            else 0
+        end as is_active,
```

Comment:
small thing: when I see is_ I assume the field will be true or false. 0 or 1 is also easily interpreted as a boolean, but it's worth thinking about: 1. the tool that will be using this (e.g. if it's Looker, 0 and 1 may be easier) and 2. naming patterns (e.g. if it's a boolean data type I could test the name to see if it starts with is_ or has_, but that test is not possible here). 

TLDR: true or false is easier to interpret because if I see  is_active==False I know better what it means than just seeing is_active==0. Small thing but worth thinking about when implementing booleans across a project

Reply:
makes sense! till be consistent with creating true vs false bool fields moving forward